[FIX] SigLIP 2 (#2699)
huggingface/blog: siglip2.md

Running inference on the models is pretty straightforward. You can copy paste the code below and run inference on a free tier Colab notebook 🚀

> [!NOTE]

> > To run inference on SigLIP 2, please install `transformers` from GitHub. (It will soon be in the latest `transformers` release)
> > `pip install git+https://github.com/huggingface/transformers@main`
> > OR
> > `pip install git+https://github.com/huggingface/transformers@v4.49.0-SigLIP-2`

Reviewer: ```suggestion
> To run inference on SigLIP 2, please install `transformers` from `main` or from this stable branch:
> `pip install git+https://github.com/huggingface/transformers@v4.49.0-SigLIP-2`
```

I think it's better to prefer the stable branch.
